sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -428,11 +428,19 @@ public void setStatus(Status status) {
 
   @Override
   public void recordException(Throwable exception) {
+    recordException(exception, null);
+  }
+
+  @Override
+  public void recordException(Throwable exception, Attributes additionalAttributes) {
     if (exception == null) {
       return;
     }
     long timestamp = clock.now();
-    Attributes.Builder attributes = Attributes.newBuilder();
+    Attributes.Builder attributes =
+        additionalAttributes != null
+            ? Attributes.newBuilder(additionalAttributes)
```

Comment:
even simpler is just `? addiontalAttributes.toBuilder()` yes?

Reply:
yes. fixed at lastest commit. thanks.